app.js

```diff
@@ -20,16 +20,12 @@ const bodyParser = require('body-parser')
 const methodOverride = require('method-override')
 const expressWinston = require("express-winston")
 const session = require("express-session")
-const RedisStore = require("connect-redis")(session)
+const PostgresStore = require("connect-session-sequelize")(session.Store)
 const responses = require("./api/responses")
 
 const app = express()
-
-if (config.redis) {
-  config.session.store = new RedisStore(config.redis)
-} else {
-  config.session.store = new session.MemoryStore()
-}
+const sequelize = require("./api/models").sequelize
+config.session.store = new PostgresStore({ db: sequelize })
```

Comment:
It's a little odd to see something called "db" given an ORM instance as they're not the same thing, but that's what they want in https://github.com/mweibel/connect-session-sequelize. There's nothing we can do about it.

This was the only confusing thing I saw in here.